coderedcms/templatetags/coderedcms_tags.py

```diff
@@ -40,6 +41,26 @@ def coderedcms_version():
 def generate_random_id():
     return ''.join(random.choice(string.ascii_letters + string.digits) for n in range(20))
 
+@register.simple_tag(takes_context=True)
+def twitter_card_image(context, page):
```

Comment:
You can scrap this function and use `og_image` function for both twitter card image and opengraph image tags. They should be using the same logic, so I think we just need one function for both.